packages/protocol/contracts/stability/Exchange.sol

```diff
@@ -98,7 +98,28 @@ contract Exchange is
   }
 
   /**
-   * @dev Exchanges sellAmount of sellToken in exchange for at least minBuyAmount of buyToken
+   * @notice Exchanges sellAmount of sellToken in exchange for at least minBuyAmount of buyToken
+   * Requires the sellAmount to have been approved to the exchange
+   * @param sellAmount The amount of sellToken the user is selling to the exchange
+   * @param minBuyAmount The minimum amount of buyToken the user has to receive for this
+   * transaction to succeed
+   * @param sellGold `true` if gold is the sell token
+   * @return The amount of buyToken that was transfered
+   * @dev This function can be frozen using the Freezable interface.
+   */
+  function sell(uint256 sellAmount, uint256 minBuyAmount, bool sellGold)
+    external
+    onlyWhenNotFrozen
+    updateBucketsIfNecessary
+    nonReentrant
+    returns (uint256)
+  {
+    return _sell(sellAmount, minBuyAmount, sellGold);
+  }
+
+  /**
+   * @notice Deprecated. Exchanges sellAmount of sellToken in exchange for at
```

Comment:
```
@dev DEPRECATED - Use `buy` or `sell`
@notice Exchanges a specific amount of one token for an unspecified amount of another.
@param sellAmount The number of tokens to send to the exchange.
@param minBuyAmount The minimum number of tokens for the exchange to send in return.
@param sellGold True if the caller is sending CELO to the exchange, false otherwise.
@return The number of tokens sent by the exchange.
@dev The caller must first have approved `sellAmount` to the exchange.
@dev This function can be frozen via the Freezable interface.```